volatility3/cli/volshell/generic.py

```diff
@@ -149,6 +149,7 @@ def construct_locals(self) -> List[Tuple[List[str], Any]]:
             (["cc", "create_configurable"], self.create_configurable),
             (["lf", "load_file"], self.load_file),
             (["rs", "run_script"], self.run_script),
+            (["re", "regex_scan"], self.regex_scan),
```

Comment:
I worry that re may conflict with the python re library?  Could shorthand be changed to another code?

Reply:
Yup! That is an extremely good point. I've swapped it for `rx` which is still somewhat RegeX related. :D I'm not too fussed what shorthand to use if you can think of a better one...!